.github/workflows/new-milestone-and-issue-in-other-repo.yml

```diff
@@ -24,6 +25,11 @@ jobs:
     steps:
       - name: Checkout the latest code
         uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - name: Download xmlstarlet and ncal
+        run: |
+          sudo apt-get install xmlstarlet ncal
```

Comment:
`cal` and `xmlstarlet` were missing [as shown here](https://github.com/stoyanK7/checkstyle/actions/runs/4015598048/jobs/6897515929#step:3:77).